DevAcademy/DevAcademyApp.swift

```diff
@@ -4,7 +4,9 @@ import SwiftUI
 struct DevAcademyApp: App {
     var body: some Scene {
         WindowGroup {
-            ContentView()
+            NavigationStack {
```

Comment:
Většinou NavigationStack schováváme až někam hlouběji. Je nějaký důvod pro použití už tady?

Reply:
Není, nebo mne nenapadá, proč jsem to sem šoupnul. Upravil jsem - vložil jsem do `PlacesScene`. Díky za tip ☺️